src/lumigo_tracer/spans_container.py

```diff
@@ -116,6 +121,8 @@ def start(self, event=None, context=None):
     def handle_timeout(self, *args):
         get_logger().info("The tracer reached the end of the timeout timer")
         to_send = [s for s in self.http_spans if s["id"] in self.http_span_ids_to_send]
+        if Configuration.send_only_if_error:
+            to_send.append(self._generate_start_span())
```

Comment:
Does the order matter? Maybe it should be the first in `to_send`?
`to_send.insert(0, self._generate_start_span())`

Reply:
it doesn't metter, and if there will be many spans it is unnecessary compute waste 🤷